docs/backend.openapi.yaml

```diff
@@ -113,6 +113,61 @@ paths:
           description: User successfully signed out
         "403":
           description: Access denied
+  
+  /chat/users/{username}/messages:
+    put:
+      summary: User sends a message
+      tags:
+        - chat
+      security:
+        - auth: []
+      parameters:
+        - $ref: "#/components/parameters/username"
+      responses:
+        "200":
+          description: Message was successfully handled
+        "403":
+          description: Access denied
+
+  /chat/messages:
+    get:
+      summary: Last 100 messages in the chat
+      tags:
+        - chat
+      security:
+        - auth: []
+      responses:
+        "200":
+          description: List of last messages
+          content:
+            application/json:
+              schema:
+                type: array
+                items: 
+                  $ref: "#/components/schemas/Message" 
+        "403": 
+          description: Access denied
+
+  /chat/messages/{id}:
```

Comment:
1. Naming it `id` means that client knows that message id is used as cursor. What if server starts using some other way to reference cursor, like for example "timestamp" ? In this case the name `id` becomes wrong.

For this reason lets name it abstractly `cursor`. This way there is no information other than its purpose is encoded.

2. As the `cursor` parameter is optional (first portion client requests without it) it makes sense to put it into a query parameter.

`GET /chat/messages/?cursor=3dsfa345s`